pkg/resources/scim_integration.go

```diff
@@ -71,37 +75,72 @@ var scimIntegrationSchema = map[string]*schema.Schema{
 	},
 }
 
-// SCIMIntegration returns a pointer to the resource representing a network policy.
 func SCIMIntegration() *schema.Resource {
 	return &schema.Resource{
 		CreateContext: CreateContextSCIMIntegration,
 		ReadContext:   ReadContextSCIMIntegration,
 		UpdateContext: UpdateContextSCIMIntegration,
 		DeleteContext: DeleteContextSCIMIntegration,
-
+		CustomizeDiff: customdiff.All(
+			BoolComputedIf("sync_password", "SYNC_PASSWORD", func(client *sdk.Client, id sdk.AccountObjectIdentifier) ([]Property, error) {
+				props, err := client.SecurityIntegrations.Describe(context.Background(), id)
+				res := make([]Property, len(props))
+				for i := range props {
+					res[i] = props[i]
+				}
+				return res, err
+			}),
+		),
 		Schema: scimIntegrationSchema,
 		Importer: &schema.ResourceImporter{
 			StateContext: schema.ImportStatePassthroughContext,
 		},
+		SchemaVersion: 1,
```

Comment:
Please add it as first prop of Resource and add interline after

Reply:
Done.